### ISSUE TITLE
Support for PandaCSS

### ISSUE THREAD

**Issue by k3nsei** [angular/angular-cli#25411]
### Command

build

### Description

[PandaCSS](https://panda-css.com/) is first CSS-in-JS solution that works in Angular. Only problem that I have with it is not perfect developer experience, because Angular CLI don't allow to override PostCSS config to make it seamless and to work out of the box. As there is support for TailwindCSS already done somehow it should be inclusive for other technologies and not to favorite one solution over the other.

Here you can find example app https://github.com/k3nsei/ng-panda-css

### Describe the solution you'd like

Angular CLI could expose function to decorate it's postcss config and read `postcss.config.js` in root directory or way to declare own postcss plugins configs in `angular.json`.

### Describe alternatives you've considered

Now I need to run separate watch command in background in parallel to ng serve and have prebuild command when building app. Those are extra steps that affecting developer experience.
